pkgs/development/libraries/qt-5/modules/qtdeclarative.nix

```diff
@@ -21,4 +23,10 @@ qtModule {
     "bin/qmlscene"
     "bin/qmltestrunner"
   ];
+  postFixup = lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
+    mv $dev/bin/qmlformat $bin/bin/qmlformat
+    mv $dev/bin/qmltyperegistrar $bin/bin/qmltyperegistrar
+    ln -s $bin/bin/qmlformat $dev/bin/qmlformat
+    ln -s $bin/bin/qmltyperegistrar $dev/bin/qmltyperegistrar
+  '';
```

Comment:
why

Reply:
I'm running a build to get the exact failure, but IIRC it's related to the fact that we can't keep hostPlatform binaries out of the $PATH because:
- QT puts `qmake` and `libQtBase` in the same package
- QT goes bananas with `propagatedBuildInputs` and there is no way to "unpropagate" or "block propagation" of dependencies at a certain point in the tree.

Reply:
I was mistaken; it is not necessary anymore.  At one point in my development process it was, but it no longer is.

- https://github.com/NixOS/nixpkgs/pull/265583